recipes/nlopt/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.4.2" %}
+
+package:
+  name: nlopt
+  version: {{ version }}
+
+source:
+  fn: nlopt-{{ version }}.tar.gz
+  url: http://ab-initio.mit.edu/nlopt/nlopt-{{ version }}.tar.gz
+  md5: d0b8f139a4acf29b76dbae69ade8ac54
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python {{PY_VER}}*  # [not win]
+    - numpy  # [not win]
+
+  run:
+    - python {{PY_VER}}*  # [not win]
```

Comment:
no, VC features do not install anything.  They only choose a particular variant when variants are available.

You need Python as a runtime dependency in order to activate the correct feature.  You don't need {{PY_VER}}* though - it is implicit.  It doesn't hurt anything to have it, though.